docs_user/adoption-attributes.adoc

```diff
@@ -149,6 +149,11 @@ ifeval::["{build}" == "downstream"]
 :telemetry: Telemetry service
 endif::[]
 
+ifeval::["{build}-{build_variant}" == "downstream-ospdo"]
+:OpenStackPreviousInstaller: director_operator
```

Comment:
Yeah,  `OpenStackPreviousInstaller` = director in the rendered downstream docs, and "TripleO" in the upstream rendered docs. This was done because downstream and upstream refer to the installer differently.

TBH I'm not sure why `OpenStackPreviousInstaller` was being used for the ifeval statement for OSPdO docs. Does the OSPdO content need its own value for the  `OpenStackPreviousInstaller`  variable in the text itself? Currently, we've just written the terms "director Operator"  and "OSPdO" as is because the content is conditionalized for OSPdO anyway. I could be misunderstanding though.

Reply:
I used OpenStackPreviousInstaller as a method to differentiate ospdo from regular director , that method could be any thing actually , as long as it preserves the logic